From: Jan Engelhardt <jengelh@inai.de>
Date: 2020-04-18 20:10:34.215071041 +0200

Under SLADE 3.1.x, I observed:

[   12s] /usr/include/wx-3.0/wx/filename.h:139: undefined reference to `wxFileName::Assign(wxString const&, wxPathFormat)'
[   12s] /usr/lib64/gcc/x86_64-suse-linux/9/../../../../x86_64-suse-linux/bin/ld: CMakeFiles/external.dir/email/wxEmailMessage.o: in function `wxEmailMessage::DoAddAttachment(wxString const&, wxString&) const':

While external.so is gone in SLADE 3.2.6, the function is still used,
and so we should add "base" to WX_LIBS regardless.

---
 cmake/unix.cmake |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Index: SLADE-3.2.6/cmake/unix.cmake
===================================================================
--- SLADE-3.2.6.orig/cmake/unix.cmake
+++ SLADE-3.2.6/cmake/unix.cmake
@@ -65,7 +65,7 @@ if (WX_GTK3)
 	set(wxWidgets_CONFIG_OPTIONS --toolkit=gtk3)
 endif (WX_GTK3)
 
-SET(WX_LIBS std aui gl stc richtext propgrid)
+SET(WX_LIBS base std aui gl stc richtext propgrid)
 if (NO_WEBVIEW)
 	SET(WX_LIBS ${WX_LIBS} html)
 else (NO_WEBVIEW)
